src/NServiceBus.Core/Features/Feature.cs

```diff
@@ -214,10 +196,7 @@ internal void ConfigureDefaults(SettingsHolder settings)
         }
     }
 
-    static string GetFeatureName(Type featureType)
-    {
-        return featureType.FullName;
-    }
+    static string GetFeatureName(Type featureType) => featureType.FullName!;
```

Comment:
I always get a little worried when I see the `!` in there, but it does seem relatively unlikely for us to hit the scenarios where `FullName` would actually be `null`...

Reply:
Yeah true. It seems though if we want to address this we'd have to touch quite a few places first instead of only here